llvm/include/llvm/Transforms/Instrumentation/SYCLITTAnnotations.h

```diff
@@ -0,0 +1,27 @@
+//===----- SYCLITTAnnotations.h - SYCL Instrumental Annotations Pass ------===//
```

Comment:
It would be great if you assign yourself as a code owner for this pass in the CODEOWNERS file.